starter-code/javascript/chronometer.js

```diff
@@ -1,40 +1,78 @@
 // Constructor
-// function Chronometer() {
+function Chronometer() {
+  this.currentTime = 0;
+  this.intervalId = 0;
+}
 
-// }
+Chronometer.prototype.startClick = function () {
+  // because of scoping, make a copy of this (otherwise we can't access this.currentTime++;)
+  var self = this;
+  self.intervalId = setInterval(function () {
+    self.currentTime++;
+    self.setTime();
+  }, 10);
+};
 
-// Chronometer.prototype.startClick = function () {
+Chronometer.prototype.setMinutes = function () {
+  // return how many minutes (from seconds total). (10 * 6000 = 60.000 miliseconds which is 1 minute)
+  return Math.floor(this.currentTime / 6000);
+};
 
-// };
+Chronometer.prototype.setSeconds = function () {
+  // return seconds left after calculating minutes. 
+  return Math.floor((this.currentTime % 6000) / 100);
+};
 
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
+Chronometer.prototype.setMilliseconds = function () {
+  // current time 100 (10*100) = 1 second. return what's left.
+  return this.currentTime % 100;
+};
 
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
+Chronometer.prototype.twoDigitsNumber = function (number) {
+  // if one digit, add a leading 0
+  if (number < 10) {
+    return "0" + number.toString();
+  } else {
+    return number.toString();
+  }
+};
 
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
+Chronometer.prototype.setTime = function () {
+  // Taking the current time and set it in minutes and seconds (2 digits)
+  var minutes = this.twoDigitsNumber(this.setMinutes());
+  var seconds = this.twoDigitsNumber(this.setSeconds());
+  var milisec = this.twoDigitsNumber(this.setMilliseconds());
+  // display seconds
+  secDec.innerHTML = seconds.substr(0, 1);
+  secUni.innerHTML = seconds.substr(1, 1);
```

Comment:
`seconds.charAt(0)` and `seconds.charAt(1)` are preferable here since you just want to get one character, and `substr()` or rather `.substring()` should be used to find sections of string. According to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr), `.substr()`, is anyway being deprecated.